contracts/cheddar_5x5_tic_tac_toe/src/internal.rs

```diff
@@ -12,10 +12,10 @@ impl Contract {
 
     /// set accuracy, max_duration need to be in range [100..3600] seconds
     #[private]
-    pub fn set_max_duration(&mut self, max_duration: u32) -> bool {
-        validate_game_duration(max_duration as u64);
-        self.max_game_duration = sec_to_nano(max_duration);
-        self.max_turn_duration = self.max_game_duration / MAX_NUM_TURNS;
+    pub fn set_max_duration(&mut self, max_duration: u64) -> bool {
```

Comment:
why do we have this function?

Reply:
It was already there in the contract. Now when I look at it we should remove it.

Reply:
It allows the owner of the contract to change the max duration of the game. Similarly to `update_fee` that changes the fee that the contract charges